spec/requests/user_pages_spec.rb

```diff
@@ -10,7 +10,7 @@
       before { visit signup_path }
 
       it { should have_content('Sign up') }
-      it { should has_title?('Sign up') }
+      it { should have_title('Expenses | Sign up') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.